catkit/hardware/zwo/ZwoCamera.py

```diff
@@ -147,14 +147,71 @@ def __capture_and_orient(self, initial_sleep, theta, fliplr):
 
         Returns
         -------
-        image : np.array of ints
-            Array of integers making up the image.
+        image : np.array of floats
+            Array of floats making up the image.
         """
 
         unflipped_image = self.__capture(initial_sleep=initial_sleep)
         image = catkit.util.rotate_and_flip_image(unflipped_image, theta, fliplr)
         return image
-    
+
+    def __capture_video(self, num_exposures, timeout):
+        """ Take a number of images.
+
+        WARNING: This func does NOT set the exposure time!
+
+        Parameters
+        ----------
+        num_exposures : int
+            The number of exposures to take.
+        timeout : Pint quantity
+            How long to wait for an image. Afterwards a timeout exception is raised.
+
+        Returns
+        -------
+        images : list of (np.array of floats)
+            All captured images in a list.
+        """
+        self.instrument.start_video_capture()
+
+        images = []
+        for i in range(num_exposures):
+            img = self.instrument.capture_video_frame(timeout=timeout.to(units.millisecond).magnitude)
```

Comment:
What happens when it times out? Does it raise or...?

Reply:
It raises a ZWO_IOError (with code 11: timeout). We might want to convert this to a RuntimeError? We can also not set the timeout. Then it will just hang and wait forever on an image that may never get there. The SDK is unclear on if a ZWO_IOError (with code 5: camera removed) will be raised if the USB disconnects during an exposure. You might know from experience if this is the case; I'm unable to test this as I cannot jank the USB cable out while taking images.

Reply:
>It raises a ZWO_IOError (with code 11: timeout). We might want to convert this to a RuntimeError? 

This is cool as is.

>We can also not set the timeout. Then it will just hang and wait forever on an image that may never get there. 

We certainly don't want to do that. The sooner something tanks out the better.

>The SDK is unclear on if a ZWO_IOError (with code 5: camera removed) will be raised if the USB disconnects during an exposure. You might know from experience if this is the case; I'm unable to test this as I cannot jank the USB cable out while taking images.

No experience there, sorry, but I don't think we care too much either, or even if we did we could do much about it anyhow.

Reply:
My main worry is that, only very sometimes, the timeout may trigger too soon. It hasn't happened yet during testing, but I'm sure it'll happen at some point. Do we want to set the timeout to an insane value (say 10-30 seconds) to ensure that the timeout only triggers when something actually went wrong?

Reply:
We could do something like 10-30s, however, it really should be a function of the exposure time. We could, however, _add_ 60s (usual default timeout) to that derived from the exp-time, i.e., to account for when the exp-time _is_ 60s.

Reply:
Some % would also work, i.e., timeout = f(exp-time)*0.2.